sdk/core/azure-core-experimental/src/main/java/module-info.java

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 module com.azure.core.experimental {
+    requires transitive com.azure.json;
```

Comment:
Is this needed? Azure core's transitive `requires` will be sufficient.

Reply:
Good catch, at this time I think the only usage is internal, so it won't need to be transitive.